lib/Pherkin/Extension/weasel_steps/widget_steps.pl

```diff
@@ -68,7 +68,7 @@
     my $label_text = $1;
     my $want_values = $2;
 
-    my $select = S->{ext_wsl}->page->find('*select', label => $label_text);
+    my $select = S->{ext_wsl}->page->find('*select', text => $label_text);
```

Comment:
The "select expander" has neither a `label` nor a `text` search criterion (see https://github.com/perl-weasel/weasel/blob/master/lib/Weasel/FindExpanders/HTML.pm#L215). 

From the commit message, it's not clear whether you think you're fixing something here or that you're changing stuff to match your preference. Could you please be specific about that?